src/amo/components/FlagReviewMenu/index.js

```diff
@@ -122,7 +124,7 @@ export class FlagReviewMenuBase extends React.Component<InternalProps> {
 
     return (
       <TooltipMenu
-        className={wasFlagged ? `${String(openerClass)}--flagged` : undefined}
+        className="flag-review-button"
```

Comment:
We have a naming convention for the classNames in our components, where they start with the component name, for example, `DropdownMenu-button` in the `DropdownMenu` component, so this should be something like `FlagReviewMenu-menu`.